src/test/java/base/TestFlow.java

```diff
@@ -0,0 +1,81 @@
+package base;
+
+import helper.BrowserFactory;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pages.*;
+
+import static helper.BrowserFactory.driver;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class TestFlow {
+
+    private static final
+    Logger logger = Logger.getLogger(BrowserFactory.class.getName());
+
+
+    @BeforeTest
+    public void createAccount() {
+        // Launch the browser
+        driver = BrowserFactory.startBrowser("chrome", "https://parabank.parasoft.com");
+        String page_title = driver.getTitle();
+
+        // Printing the title of the Page
+        logger.info("Page Title: " + page_title);
+
+        // Create a new account
+        CreateAccount createAccount = PageFactory.initElements(driver, CreateAccount.class);
+        createAccount.createAccount("virat", "mishra", "mandhana", "kanpur", "UP", "586665", "9876565655", "1234", "ViratMishra", "Virat@1234", "Virat@1234");
+
+    }
+    @Test(priority = 0)
+    public void loginAccount(){
+        CreateAccount loginAccountParaBank  = PageFactory.initElements(driver, CreateAccount.class);
+        loginAccountParaBank.loginAccount("ViratMishra", "Virat@1234");
+    }
+    @Test(priority = 1)
+    public void openAccount() throws InterruptedException {
+        Account account = PageFactory.initElements(driver, Account.class);
```

Comment:
Create a separate class that returns object of all the classes. Instead of instantiate all the POM classes again and again in test methods.